lib/internal/modules/package_json_reader.js

```diff
@@ -185,8 +185,8 @@ function getPackageScopeConfig(resolved) {
  * @param {URL} url - The URL to get the package type for.
  */
 function getPackageType(url) {
-  // TODO(@anonrig): Write a C++ function that returns only "type".
-  return getPackageScopeConfig(url).type;
+  const type = modulesBinding.getPackageType(`${url}`);
```

Comment:
Can you add this method to typings folder as well?

Reply:
ops, sorry my bad I forgot about that :bow: 

done, thanks :slightly_smiling_face::+1: